lpc55_sign_bin/src/main.rs

```diff
@@ -321,6 +408,74 @@ fn main() -> Result<()> {
             let out = lpc55_sign::signed_image::remove_image_signature(image)?;
             std::fs::write(dst_img, out)?;
         }
+        Command::DebugCredential {
+            root_certs_cfg,
+            root_key,
+            debug_key,
+            uuid,
+            vendor_usage,
+            beacon,
+            debug_settings_cfg,
+            dest_dc,
+        } => {
+            let root_certs_cfg = std::fs::read_to_string(root_certs_cfg)?;
+            let root_certs_cfg: CertConfig = toml::from_str(&root_certs_cfg)?;
+            let root_certs =
+                read_certs(&root_certs_cfg.root_certs).context("Reading root certificates")?;
+
+            let root_private_key =
+                read_rsa_private_key(&root_key).context("Reading root private key")?;
+
+            let debug_private_key =
+                read_rsa_private_key(&debug_key).context("Reading debug private key")?;
+            let debug_public_key = debug_private_key.to_public_key();
+
+            let uuid = match uuid {
+                Some(x) => <[u8; 16]>::from_hex(x)?,
+                None => [0; 16],
+            };
+
+            let debug_settings = std::fs::read_to_string(&debug_settings_cfg)?;
```

Comment:
I notice you're not `context`-ing the `read_to_string` calls, how's its error message if you typo the filename?